recipe-server/client/control/tests/components/extensions/test_ExtensionSelect.js

```diff
@@ -36,4 +36,42 @@ describe('<ExtensionSelect>', () => {
     const placeholderStyle = placeholderElement.get(0).style;
     expect(placeholderStyle.display).toBe('block');
   });
+
+  it('should display its inherited value prop', () => {
+    const wrapper = mount(wrapMockStore(
+      <ExtensionSelect
+        {...props}
+        extensions={fromJS([{ xpi: '1', name: 'one' }, { xpi: '2', name: 'two' }])}
+        value="2"
+      />,
+    ));
+
+    expect(wrapper.find('.ant-select-selection-selected-value').text()).toBe('two');
+  });
+
+  it('should fire an onChange event appropriately', async () => {
+    let selected;
+    const wrapper = mount(wrapMockStore(
+      <ExtensionSelect
+        {...props}
+        extensions={
+          fromJS([{ xpi: '1', name: 'one' }, { xpi: '2', name: 'two' }])
+        }
+        onChange={val => {
+          selected = val;
+        }}
+      />,
+    ));
+
+    wrapper.find('.ant-select-selection__placeholder').simulate('click');
+    expect(wrapper.find(Select).props().children.size).toBe(2);
```

Comment:
Is children in this line the elements of the dropdown? Would it work to simulate('click') on one of them?

Reply:
After playing with this a bit, I'm still at a wall.

- The 'children' reflect the options passed in, however selecting one (`.get(0)`) returns some sort of wrapper which does _not_ allow `simulate`ing events
- The children have `title` and `value` props, but searching the wrapper (`wrapper.find("[type='whatever']")` turns up empty
  - I suspect this is because Ant is rendering the dropdown menu someplace else on the page. For instance, if you look at the DOM, the dropdown menu is _not_ nested under the actual select box.
- Searching for the exact Option class names (`wrapper.find('.ant-select-dropdown-menu-item')`) also turns up nothing

At this point, I think I will have to dig into how Ant/rc-select performs their tests on this component.